src/misc/setup.cpp

```diff
@@ -1135,13 +1135,29 @@ Verbosity Config::GetStartupVerbosity() const
 	return Verbosity::High;
 }
 
+bool CommandLine::IsOption(const std::string &name, cmd_it &it) {
+	if (!(FindEntry(("/" + name).c_str(), it, false)) &&
+	    !(FindEntry(("-" + name).c_str(), it, false)) &&
+	    !(FindEntry(("--" + name).c_str(), it, false)))
+		return false;
+	return true;
```

Comment:
If it's `not and not and not`, then return not.

If we flip the double negatives, we get "If it's `true or true or true` then return true.

So I think this can become easier to read, with:

```c++
return FindEntry("/" + name, it, false) ||
	    FindEntry("-" + name, it, false) ||
	    (FindEntry("--" + name, it, false);
```

Reply:
Nice simplification to the code. Done.